test/e2e/grants/auth_code.grant.spec.ts

```diff
@@ -606,13 +662,18 @@ describe("authorization_code grant", () => {
       request = new OAuthRequest({
         body: {
           token: authorizationCode,
+          client_id: client.id,
```

Comment:
I added the client Id as the token was not really revoked before so the test did not actually cover the happy path.
The 200 response was only due to the silent failure implementation as defined in the spec.

> // Silently ignore - per RFC 7009, invalid tokens should not cause error responses
    // @see https://datatracker.ietf.org/doc/html/rfc7009#section-2.2